cpp-throughput/throughput_scaling.cpp

```diff
@@ -114,38 +111,38 @@ void run_throughput(std::ofstream& timing_file,
 
     timing_file << std::flush;
     MPI_Barrier(MPI_COMM_WORLD);
-    log_data(context, LLDebug, "run_throughput() method completed and all tensor times written to file");
+    log_data(context, LLDebug, "run_throughput() method completed");
     return;
 }
 
 int main(int argc, char* argv[]) {
-    std::string context("Throughput Scaling Tests");
     MPI_Init(&argc, &argv);
 
     int rank;
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
-    log_data(context, LLDebug, "rank: " + std::to_string(rank) + " initiated");
+    std::string context("Throughput Scaling Tests Rank " + std::to_string(rank));
+    log_data(context, LLDebug, "Rank initialized");
 
-    if(argc==1)
+    if(argc==1) {
+        log_error(context, LLInfo, "Error thrown");
```

Comment:
Pull out string below to a constant, and use that in the log message. "Error thrown" isn't terribly useful as a log message.

Reply:
thank u for catching that!